x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/action_connector_form/create_connector_flyout/index.test.tsx

```diff
@@ -168,12 +186,14 @@ describe('CreateConnectorFlyout', () => {
 
   it('disables the buttons when there are error on the form', async () => {
     const { getByTestId } = appMockRenderer.render(
-      <CreateConnectorFlyout
-        actionTypeRegistry={actionTypeRegistry}
-        onClose={onClose}
-        onConnectorCreated={onConnectorCreated}
-        onTestConnector={onTestConnector}
-      />
+      <Router history={history}>
+        <CreateConnectorFlyout
+          actionTypeRegistry={actionTypeRegistry}
+          onClose={onClose}
+          onConnectorCreated={onConnectorCreated}
+          onTestConnector={onTestConnector}
+        />
+      </Router>
```

Comment:
There is a lot of copy-paste of the same code. Could you please introduce a helper function 

```typescript
const renderFlyout = () => {
   appMockRenderer.render(
      <Router history={history}>
        <CreateConnectorFlyout
          actionTypeRegistry={actionTypeRegistry}
          onClose={onClose}
          onConnectorCreated={onConnectorCreated}
          onTestConnector={onTestConnector}
        />
      </Router>
    );
}
```

so then you could simply call 
```typescript
const { getByTestId } = renderFlyout();
```